README.md

```diff
@@ -1,7 +1,7 @@
 # Unleash::Client
 
-[![Build Status](https://travis-ci.org/Unleash/unleash-client-ruby.svg?branch=master)](https://travis-ci.org/Unleash/unleash-client-ruby)
-[![Coverage Status](https://coveralls.io/repos/github/Unleash/unleash-client-ruby/badge.svg?branch=master)](https://coveralls.io/github/Unleash/unleash-client-ruby?branch=master)
+[![Build Status](https://travis-ci.org/Unleash/unleash-client-ruby.svg?branch=main)](https://travis-ci.org/Unleash/unleash-client-ruby)
```

Comment:
```suggestion
![Build Status](https://github.com/unleash/unleash-client-ruby/actions/workflows/pull_request.yml/badge.svg?branch=main)]
```

Reply:
@rarruda Is this the required change for the build status or is there more that needs to be done?

Reply:
@thomasheartman This is perfect!